packages/vats/src/core/chain-behaviors.js

```diff
@@ -146,9 +147,24 @@ export const bridgeProvisioner = async ({
     async fromBridge(_srcID, obj) {
```

Comment:
how does `fromBridge` here relate to the one in `provisionPool.js`? I'd like to see compare/contrast comments on each.